Adyen/adyen-magento2
# [PW-7179] - Adjust vault tokenization functionality

**Description**
This PR will adjust some functionality used for tokenization. Among others, the `AdyenUiComponentProvider` will be re-introduced and for now amazon pay and apple pay will not be usable.

**Tested scenarios**

For **CardOnFile**:
  * Initial Google Pay payment
  * Tokenised Google Pay payment
  * Card payment
  * Tokenised card payment
  * Delete tokens from my account page

For **Subscription** and **UnscheduledCardOnFile**:
  * Initial Google Pay payment
  * Check token in `vault_payment_token` created with the respective tokenType
  * Initial card payment
  * Check token in `vault_payment_token` created with the respective tokenType
  * Initial Paypal payment
  * Check token in `vault_payment_token` created with the respective tokenType
